action-server/tests/actions/test_question_answering_form.py

```diff
@@ -0,0 +1,226 @@
+from unittest.mock import MagicMock, patch
+
+from rasa_sdk.events import Form, SlotSet
+from rasa_sdk.forms import REQUESTED_SLOT
+
+from covidflow.actions.answers import QuestionAnsweringResponse, QuestionAnsweringStatus
+from covidflow.actions.question_answering_form import (
+    ANSWERS_SLOT,
+    ASKED_QUESTION_SLOT,
+    FEEDBACK_SLOT,
+    FORM_NAME,
+    QUESTION_SLOT,
+    STATUS_SLOT,
+    QuestionAnsweringForm,
+)
+
+from .form_helper import FormTestCase
+
+
+def QuestionAnsweringResponseMock(*args, **kwargs):
+    mock = MagicMock(*args, **kwargs)
+
+    async def mock_coroutine(*args, **kwargs):
+        return mock(*args, **kwargs)
+
+    mock_coroutine.mock = mock
+    return mock_coroutine
+
+
+QUESTION = "What is covid?"
+ANSWERS = [
+    "It's a virus!",
+    "It's the greatest plea since the plague!",
+    "No, it's SU-PER BAD!",
+]
+
+SUCCESS_RESULT = QuestionAnsweringResponse(
+    answers=ANSWERS, status=QuestionAnsweringStatus.SUCCESS
+)
+FAILURE_RESULT = QuestionAnsweringResponse(status=QuestionAnsweringStatus.FAILURE)
+OUT_OF_DISTRIBUTION_RESULT = QuestionAnsweringResponse(
+    status=QuestionAnsweringStatus.OUT_OF_DISTRIBUTION
+)
+
+
+FULL_RESULT_SUCCESS = {
+    "question": QUESTION,
+    "answers": ANSWERS,
+    "status": QuestionAnsweringStatus.SUCCESS,
+    "feedback": True,
+}
```

Comment:
Do we have constants for these properties?

Reply:
No but I will add them